app/src/components/buttons/toggleButton/toggleButton.scss

```diff
@@ -53,3 +52,35 @@
     background: $COLOR--gray-91;
   }
 }
+
+.separated {
+  display: flex;
+  height: 23px;
+
+  .button-item {
+    display: inline-block;
+    float: none;
+    height: 100%;
+    border: none;
+    border-radius: 100px;
+    margin: 0 6px;
+    padding: 5px 15px;
+    color: $COLOR--gray-47;
+    background-color: $COLOR--white-four;
+    font-family: $FONT-SEMIBOLD;
+    font-size: 12px;
+    opacity: 0.5;
+
+    &.active {
+      color: $COLOR--black-2;
+      background-color: $COLOR--white-two;
+      box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.2);
+      opacity: 1;
+      cursor: default;
+    }
+
+    span {
```

Comment:
Please avoid to use tag selectors. Markup and styles should be as independent as possible.

Reply:
Fixed